crates/lib/kajiya/src/renderers/csgi.rs

```diff
@@ -261,6 +261,7 @@ impl CsgiRenderer {
         //
         // Either is going to leave the GPU quite under-utilized though, and would be a nice fit for async compute.
 
+        #[cfg(feature = "ray-tracing")]
```

Comment:
This is a bit messy; ideally the entire csgi renderer should be unused when ray-tracing is disabled. I suspect you're only after the resources that it creates -- and I also suspect it's those resources that are too large on your macbook, so that you ended up clamping texture size.

We can side-step all that by not disabling the whole renderer, and only creating placeholder resources for where it needs to be passed in -- something like a black 1x1x1 volume.